message.go

```diff
@@ -380,7 +485,31 @@ func (this *CommandMsg) messagePage(page string, server *Server) {
 	return
 }
 
+func (this *CommandMsg) messageGroups(groups []string, server *Server) {
+	msg, err := this.formatMessage()
+	if err != nil {
+		return
+	}
+
+	server.Stats.LogPageMessage()
```

Comment:
This should be calling a `LogGroupMessage()` function and should probably take in the length of the group array. so something like: `LogGroupMessage(len(groups))`